json_text.js

```diff
@@ -459,7 +455,7 @@ arrPopups = [
                "<li>LerNetz</li>" +
                "</ul>" +
                "Konzept und Umsetzung: LUDOK<br>" +
-               "Fachautoren: Meltem Kutlar Joss, Lara Stucki und Zoe Roth<br>" +
+               "Fachautoren: Meltem Kutlar Joss, Lara Stucki, Ron Kappeler und Zoe Roth<br>" +
                "Grafiker und Softwareentwickler: <a href=\"https://www.dr-carl.com\" target=\"_blank\">Dr. Carl GmbH</a>" +
                "<br><br>" +
                "Der Inhalt entspricht dem Wissensstand von 2020." +

```

Comment:
Die Jahreszahlen im Impressum zum Wissensstand könnten (in allen 4 Sprachen) wahrscheinlich auch auf 2021 oder sogar 2022 angepasst werden, oder?

Reply:
@ursjoss Stimmt, das macht Sinn das auf 2022 anzupassen. Ich wolle das soeben gleich machen, scheine aber doch mit GitHub noch nicht so ganz klar zu kommen, wo ich diese Änderung vollziehen kann.